test/powershell/Language/Operators/ReplaceOperator.Tests.ps1

```diff
@@ -11,6 +11,12 @@ Describe "Replace Operator" -Tags CI {
             $res | Should -BeExactly "image.jpg"
         }
 
+        It "Replace operator can convert an substitution object to string" {
+            $substitution = Get-Process -Name pwsh
```

Comment:
`Get-Process -Name pwsh` may returns an array, and that may cause the test to fail.

```suggestion
            $substitution = Get-Process -Id $pid
```